src/components/index/item/sidebarPresenter.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+
+import s from "./sidebar.module.scss";
+import HeadingPresenter from "components/heading/headingPresenter";
+import { onclick, tags } from "./sidebarContainer";
+
+const SideBar: React.FC = () => {
+  return (
+    <aside>
+      <HeadingPresenter
+        heading={"모두에게 하고픈 말,"}
+        explanation={"무엇인가요?"}
+      />
+      <div className={s.inputWrapper}>
+        <input className={s.title} placeholder="제목을 입력하세요." />
+        <button className={s.tagBtn}>
+          태그
+          <ul>
+            {React.Children.map(tags, (child) => (
+              <li onClick={onclick}>{child}</li>
```

Comment:
```suggestion
              <li onClick={onclick}>&num;{child}</li>
```
다음과 같이하고, `src/components/index/item/sidebarContainer.ts`에서 `tags`안의 값들에서 #을 제외하는 것이 보기 깔끔하지않을까 싶습니다.
`&num;`의 경우 `#`의 html entity입니다.

Reply:
오오 좋군요 중복되는 요소들은 최대한 배제해야죠